lambdas/db-migration/src/index.ts

```diff
@@ -0,0 +1,52 @@
+import Knex from 'knex';
+import * as path from 'path';
+
+export type Command = 'latest';
+
+export interface HandlerEvent {
+  command?: Command,
+  env?: NodeJS.ProcessEnv
+}
+
+const getRequiredEnvVar = (name: string, env: NodeJS.ProcessEnv): string => {
+  const value = env?.[name];
+
+  if (value) return value;
+
+  throw new Error(`The ${name} environment variable must be set`);
+};
+
+const getConnectionConfig = (env: NodeJS.ProcessEnv): Knex.PgConnectionConfig => ({
+  host: getRequiredEnvVar('PG_HOST', env),
+  user: getRequiredEnvVar('PG_USER', env),
+  // TODO Get this value from secrets manager
```

Comment:
Is there anything preventing us from doing this now?

Reply:
I would expect that we don't have the database TF sorted yet, and in some cases that's generated and put in SM by AWS. 

Agree that we should do this asap.

Reply:
Yeah, I was waiting for the database TF stuff. To some extent, this is just a placeholder.